sql/sql_udf.cc

```diff
@@ -206,6 +206,15 @@ void udf_init()
     DBUG_PRINT("info",("init udf record"));
     LEX_CSTRING name;
     name.str=get_field(&mem, table->field[0]);
+
+    // Check the name.str is NULL or not.
+    if (name.str == nullptr)
+    {
+      sql_print_error("Invalid row in mysql.func table for function '%.64s'",
+                      name.str);
```

Comment:
If the name.str ls nullptr it can't be used in sql_print_error. If its empty "" always then there's not much point in displaying it.